driver_guided_robot/launch/kinect_driver_only.launch.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2020 Zoltán Rési
```

Comment:
Do we really need this file here? Or do we only need it on the follow_endpoint branch?